build.gradle

```diff
@@ -195,16 +206,15 @@ tasks.withType(JavaCompile).configureEach {
 }
 
 // Register different Skript testing tasks
-createTestTask('quickTest', 'src/test/skript/environments/' + latestEnv, false, latestJava, false)
-createTestTask('skriptTestJava17', 'src/test/skript/environments/java17', false, latestJava, false)
-createTestTask('skriptTestJava8', 'src/test/skript/environments/java8', false, oldestJava, false)
-createTestTask('skriptTestDev', 'src/test/skript/environments/' + (project.property('testEnv') == null
-	? latestEnv : project.property('testEnv') + '.json'), true, Integer.parseInt(project.property('testEnvJavaVersion') == null
-	? latestJava : project.property('testEnvJavaVersion')), false)
+String env = project.property('testEnv') == null ? latestEnv : project.property('testEnv') + '.json'
+int envJava = project.property('testEnvJavaVersion') == null ? latestJava : Integer.parseInt(project.property('testEnvJavaVersion') as String)
+createTestTask('quickTest', 'src/test/skript/environments/' + latestEnv, latestJava)
```

Comment:
Can `src/test/skript/environments/` be set to its own variable instead of it being duplicated many times